Packs/PAN-OS/Scripts/PanoramaSearchPolicy/PanoramaSearchPolicy.py

```diff
@@ -0,0 +1,67 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from typing import Any
+import traceback
+
+
+''' STANDALONE FUNCTION '''
+
+
+def panorama_search_policy_command(args: Dict[str, Any]) -> list:
+    """Takes rule_name and returns output of !pan-os-list-rules or null message.
+
+    Args:
+        args (Dict[str, Any]): Demisto.args() object.
+
+    Returns:
+        List: Return from !pan-os-list-rules or message.
+    """
+    rule_name = args.get('rule_name', None)
```

Comment:
```suggestion
    rule_name = args.get('rule_name')
```

you can also use `args["rule_name"]` because its required


redundant